### ISSUE TITLE
IPTV Extreme [Android]

### ISSUE THREAD

**Issue by Nzzyme** [AdguardTeam/AdguardFilters#186643]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Android

### AdGuard version

4.6.1

### What type of problem have you encountered?

Website or app doesn't work properly

### Which browser(s) do you use?

the problem is with app, not a site

### Which device do you use?

Mobile

### Where is the problem encountered?

https://play.google.com/store/apps/details?id=com.pecana.iptvextreme

### Ad Blocking

AdGuard Base filter, AdGuard Mobile Ads filter

### Privacy

AdGuard URL Tracking filter, AdGuard Tracking Protection filter

### Social Widgets

AdGuard Social Media filter

### Annoyances

AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Other Annoyances filter, AdGuard Widgets filter, Adblock Warning Removal List

### Security

Online Malicious URL Blocklist

### DNS filters

AdGuard DNS filter

### Add your comment and screenshots

AdGuard 'Ad blocker' gets detected and unable to use the app as soon as you launch it.

### Privacy

- [X] I agree to follow this condition

**Comment by Alex-302** [comment]
It just detects VPN interface.

**Comment by Nzzyme** [comment]
> It just detects VPN interface.

So is there nothing that can be done about it apart from not running AG?

**Comment by Alex-302** [comment]
Yes. But you can create a feature request like "Pause protection when *** app is running". I think it is possible, like with Samsung Pay in low-level settings.
